src/hardware/ne2000.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include <cstdio>
 #include <cstring>
+#include <stdarg.h>
```

Comment:
Suggest switching to `<cstdsrg>` (and moving up to maintain alphabetic order).

Reply:
Is the plan to change that [everywhere `stdarg.h` is included](https://github.com/dosbox-staging/dosbox-staging/search?q=stdarg.h) or just for new code?

Reply:
Done.